docs/extension.rst

```diff
@@ -4,13 +4,14 @@ Visual Studio Code Extension
 
 Solang has
 `language server <https://en.wikipedia.org/wiki/Language_Server_Protocol>`_ built
-into the executable, which can be used by the Visual Studio Code extension. This
+into the executable, which can be used by the Visual Studio Code extension, or by
+any editor that can use a lsp language server. The Visual Studio Code
 extension provides the following:
 
-1. Syntax highlighting
+1. Syntax highlighting.
 2. Compiler warnings and errors are displayed in the problems tab and marked
```

Comment:
```suggestion
2. Compiler warnings and errors displayed in the problems tab and marked
```
The items in an enumeration must maintain parallelism. If your first item does not contain an action verb, the second one should not have it too.